src/us/kbase/workspace/kbase/ArgUtils.java

```diff
@@ -407,14 +443,14 @@ public static List<Tuple7<String, String, String, Long, String, String, Long>> w
 	 * speed up the updates, but the drawback is that if the external update fails for any object,
 	 * all the objects that required updates for that system will be marked as having a failed
 	 * update. Has no effect if the permissions handler is not present.
-	 * @param logObjects if true, log the object ref and type.
+	 * @param objectInfoAsClass return the object information as a Class rather than a Tuple.
```

Comment:
The answer will probably reveal itself further down the PR, but is this a new (and very sensible) feature? Is `log_objects` being silently disposed of, like the bodies of so many of your previous victims?

Reply:
It turned out that every method that called this method set logObjects to true, so there was no need for the argument. The class vs tuple thing should be clear when you read the spec changes (you might want to do that first in fact)